.github/workflows/docker.yml

```diff
@@ -0,0 +1,31 @@
+name: Build and Push Docker Image
+
+on:
+  schedule:
+    # run every night at midnight
+    - cron: '0 0 * * *'
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ 'alpine', 'archlinux', 'debian', 'fedora', 'gentoo', 'opensuse' ]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
@Alexays FYI for this to work you'd need to add your dockerhub username and password as secrets to the repo